instagram_scraper/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .instagram_scraper import main
+
+__name__ = "Instagram Scraper"
+__author__ = "Meet Mangukiya <meetmangukiya98@gmail.com>"
+__version__ = '1.0.0'
```

Comment:
Not a stable API yet, `0.1.0` please.

Reply:
Import `scrape_instagram_tag`, and `scrape_instagram` here... `from instagram_scraper.instagram_scraper import ...` is too long.